x/admin/keeper/keeper.go

```diff
@@ -100,3 +100,21 @@ func (k Keeper) GetAdminAccounts(ctx sdk.Context) []*types.AdminAccount {
 	}
 	return accounts
 }
+
+func (k Keeper) SetParams(ctx sdk.Context, params *types.Params) {
+	store := ctx.KVStore(k.storeKey)
+	store.Set(types.ParamsStorePrefix, k.cdc.MustMarshal(params))
+}
+
+func (k Keeper) GetParams(ctx sdk.Context) *types.Params {
+	defaultSubmitProposalFee := sdk.NewUintFromString("500000000000000000000")
```

Comment:
@timlind the value seems to be equal to 500 rowan instead of 5000 is it intentional?